microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -306,23 +302,37 @@ Congratulations, you built your first WebView2 app!
 <!-- ====================================================================== -->
 ## Status of WebView2 functionality on WinUI 2 (UWP)
 
-The WebView2 WinUI 2 (UWP) control is in development.  The following features haven't been implemented, or have issues:
+The WebView2 WinUI 2 (UWP) control is in development.  The following features haven't been implemented, or are disabled due to pending work:
 
-*  Download UI.
-   *  Although the Download UI feature currently doesn't work, this feature might show up automatically in your app's UI after the issue is fixed.  To maintain compatibility, you should manually disable the Download UI feature by intercepting the [download starting event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2downloadstartingeventargs).
+*  Downloads.
+   *  Downloading files is only available in preview via the Edge Dev channel until Edge 104; moreover, it is disabled by default. To enable, please use the command line switch `edge-webview-optional-enable-uwp-regular-downloads`. There are a couple of known limitations to the current downloads behavior for WebView2 in UWP:
+        * The downloads hub is currently disabled.
+        * Depending on the user's OS, downloaded files will be downloaded to different folders. For Windows 11, the downloaded files will appear in a subfolder with the app package's name in the Downloads folder. For OS's below Windows 11, the downloaded files will appear in a subfolder named WebView2Downloads within this app package's subfolder in the Downloads folder.
+        * Additionally, saving files via Save As is functional and enabled for UWP WebView2. The files will be saved in the respective folder the user selects.
 *  Autofill UI.
-*  File Picker Dialog.
-*  Background Audio.
 *  Print to PDF.
-*  Print Preview.
-*  Adding COM Objects (WinRT `AddHostObject`).
-*  Playready DRM.
-*  Service Workers on Windows devices before 20H2.
+*  Default printing is disabled, however developers may work around this by using CapturePreviewAsync (though this only captures the current viewport).
 
 Package summary:
 * [Overview of the NuGet package for Microsoft.UI.Xaml](https://www.nuget.org/packages/Microsoft.UI.Xaml/)
 
 
+## WinUI WebView2 special considerations
+
+### SmartScreen
+WebView2 uses SmartScreen to ensure your WebView2 apps stay secure. If you would like to disable this, you may do so via environment variable.
+
+* `Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--disable-features=msSmartScreenProtection");`
+    *  Note this must be set prior to CoreWebView2Environment creation (first time WV2.Source set or EnsureCoreWebView2Async() called).
```

Comment:
@jasonstephen15 @adrastogi - please clarify which meaning is intended:

This environment variable must be set prior to `CoreWebView2` creation; this environment variable must be set before the first time your app sets `WV2.Source` or calls `EnsureCoreWebView2Async`.

This environment variable must be set prior to `CoreWebView2` creation; this environment variable must be set when your app first sets `WV2.Source` or calls `EnsureCoreWebView2Async`.

the clarification needs to be applied to both articles

Reply:
This environment variable must be set prior to `CoreWebView2` creation, which occurs when the [WebView2.Source property](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.webview2.source) is set or the [WebView2.EnsureCoreWebView2Async method](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.webview2.ensurecorewebview2async) is called.